docs/workflow/code-review.md

```diff
@@ -0,0 +1,103 @@
+# Code review
+
+- [How to submit code for review](#how-to-submit-code-for-review)
+- [How to do a code review](#how-to-do-a-code-review)
+
+## How to submit code for review
+
+1. [Pick an issue to work on](./issue-management.md#how-to-pick-an-issue)
+
+2. Check out a new branch from the latest `main` branch
+    ```bash
+    git checkout -b [branch-name]
+    ```
+
+3. Solve the issue
+
+    > If you've made a lot of changes (>300 LOC), split the changes into multiple PRs - TBA how
```

Comment:
you can either split the changes manually, use `git cherry-pick` if you were smart and making smaller commits, or do some magic with `git stash` and `git apply`. I general it's a good idea to commit and push often, this way you can check if you're over the limit already.

Reply:
@Skidle can we pls add it to the manual as well?

Reply:
will do in a follow-up https://github.com/metisova/sturdy-lamp/issues/26